### DISCUSSION TITLE
Multiple streams with the same payer and payee

### DISCUSSION THREAD

**Discussion by lapnd** [sablier-labs/evm-monorepo#738] [Q&A] [answered]
Hi,
Does Sablier support multiple streams with the same payer and payee address?
Let say, a service provider like google provides multiple services such as gmail, google docs, google sheet. And let assume that user has to pay to use these services. User may user multiple services or may be single service but multiple instances(like pay for gmail services with different domains).
In these case, user has to establish multiple stream. Each stream/service , or each stream/service instance. In both cases, it is the same payer and payee addreess. I wonder if Salbier support these cases

**Comment by PaulRBerg** [comment, marked as answer]
Hi @lapnd, thanks for your question! People like you who are steering the future development of Sablier.

Yes, with Sablier V2, it is possible to create multiple streams with the same sender and the same recipient.

Are you looking to integrate Sablier in your project, or build an app on top? If yes, here are some relevant links:

- [Docs](https://docs.sablier.com/)
- [V2 Integration Template](https://github.com/sablier-labs/sablier-v2-integration-template)
- [V2 Sandbox](https://github.com/sablier-labs/v2-sandbox)
